libr/anal/bb.c

```diff
@@ -81,6 +81,7 @@ R_API int r_anal_bb(RAnal *anal, RAnalBlock *bb, ut64 addr, ut8 *buf, ut64 len,
 			break;
 		}
 		if (oplen < 1) {
+			r_anal_op_free (op);
```

Comment:
replace this and the next line with a `goto beach` please, there is already the same code there.